webapp/client/src/pages/SubmitAcknowledgePage.test.js

```diff
@@ -3,28 +3,39 @@ import { render, screen } from "@testing-library/react";
 import SubmitAcknowledgePage from "./SubmitAcknowledgePage";
 import { I18nextProvider } from "react-i18next";
 import i18n from "i18next";
+import { fireEvent } from "@testing-library/dom";
 
 const MOCK_PROPS = {
   prevUrl: "/some/prev/path",
   logoutUrl: "/some/link/path",
 };
 
 describe("SubmitAcknowledgePage", () => {
-  it("should link to the previous page", () => {
+  beforeEach(() => {
     render(<SubmitAcknowledgePage {...MOCK_PROPS} />);
+  });
+
+  it("should link to the previous page", () => {
     expect(screen.getByText("Zurück").closest("a")).toHaveAttribute(
       "href",
       expect.stringContaining(MOCK_PROPS.prevUrl)
     );
   });
 
   it("should link to logout in text", () => {
-    render(<SubmitAcknowledgePage {...MOCK_PROPS} />);
     expect(screen.getByText("Abmelden").closest("a")).toHaveAttribute(
       "href",
       expect.stringContaining(MOCK_PROPS.logoutUrl)
     );
   });
+
+  it("should render facebook icon", () => {
+    expect(screen.getByLabelText("facebook")).toBeInTheDocument();
+  });
+
+  it("should render email icon", () => {
+    expect(screen.getByLabelText("email")).toBeInTheDocument();
+  });
```

Comment:
Because the exact url for facebook and whatsapp and the mailto for the email are built within the ShareIcons page. If we want to test the links for correctness we would do a simply copy&paste like:
`        expect(screen.getByLabelText("facebook")).toHaveAttribute(
            "href",
            expect.stringContaining(`https://www.facebook.com/sharer/sharer.php?u=${MOCK_PROPS_DEFAULT.promoteUrl}&quote=${MOCK_PROPS_DEFAULT.shareText}`)
        );`

This would be having two code places with url in strings. wdyt?

Reply:
We test the url in the integration tests. So, I am fine with that.